MainModule/Server/Core/Functions.lua

```diff
@@ -958,16 +1081,16 @@ return function(Vargs)
 
 		Shutdown = function(reason)
 			if not Core.PanicMode then
-				Functions.Message("SYSTEM MESSAGE", "Shutting down...", service.Players:GetPlayers(), false, 5)
+				Functions.Message("Server Shutdown", "The server is shutting down...", service.Players:GetPlayers(), false, 5)
```

Comment:
I actually think this should be named system message.

Because like when the game has a custom name for system messages it would be
```
Example Name

This server is shutting down...
```

Which would be really cool

You should change `"Server Shutdown"` to `Settings.SystemTitle`

Reply:
SystemTitle is used mostly for :sm

Reply:
SystemTitle is used mostly for :sm